run_bot.sh

```
@@ -590,6 +590,151 @@ test_recording() {
+        meet-teams-bot 2>&1 | tee "$log_file" &
+    
+    local docker_pid=$!
+    
+    # Wait 2 minutes before sending API stop request
+    print_info "⏰ Waiting 2 minutes before sending API stop request..."
+    sleep 120
+    
+    # Send API stop request
+    print_info "🛑 Sending API stop request..."
+    local api_response
+    api_response=$(docker exec $(docker ps -q --filter ancestor=meet-teams-bot) curl -s -X POST http://localhost:8080/stop_record \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)